### ISSUE TITLE
DF Template Enhancement: Template target Snaps to intersection on spell placement

### ISSUE THREAD

**Issue by KammaKhazi** [flamewave000/dragonflagon-fvtt#323]
Hi there, I noticed two problems since the update to separate DF Template enhancement module:-

~~1) the templates seem to target based on token scale rather than grid dimensions. In case where I've scaled up tokens up due to smaller art work this becomes an issue. I've attached a screen shot example with my module settings. My pc's scale is set to 1.4 but is only 1x1 grid token but is being targeted from what I can see based on his scale value.~~ (Tracked in #325)

2) I noticed a new issue when using the module in conjunction with the latest version of MIDI QOL (with the use DF for template targeting option selected in Midi-QOL). Now when casting spells with associated targeting templates, the templates are being forced to grid intersection snap even though you can see in the screenshot that option is off. This wasn't a problem in previous DF QOL version. And this enforce intersection snap bug doesn't occur if I manually add a template top the canvas from the left hand menu.

Many thanks for all your hard work on these modules.
![TemplateTargettingIssue](https://user-images.githubusercontent.com/81476445/157083675-6f39d25d-4ce8-4a50-aeb6-fbbb4e7092e9.jpg)

**Comment by flamewave000** [comment]
So for your first issue, I'll have to look into it and see why the image scale is affecting the token's width/height on the grid.

The second issue, the developer for MIDI had put that option in to allow DF QoL to handle templates, but they are likely looking for DF QoL, but now that I moved the template features to a new module, I'm not sure how their integration will work.

**Comment by flamewave000** [comment]
Just looking at Tim's code, you might be able to get away with it by keeping DF QoL enabled in your game as his MIDI module is looking to see if it is enabled to know if it should defer template targeting.

**Comment by flamewave000** [comment]
I've opened a Merge Request with the MIDI project that will fix that issue.
https://gitlab.com/tposney/midi-qol/-/merge_requests/64

**Comment by KammaKhazi** [comment]
Hi many thanks for the super fast response and looking into it for me. 

Re: my second issue I think midi was obfuscating my problem a bit. I have now cloned my world and removed midi-qol etc for further testing. I probably could of described it better....
On left hand menu templates don't snap to grid intersections, but when templates are placed by a spell (I used SRD Fireball as a test) they are now snapping to grid intersections (despite the DF Template Enhancement option being off).
kind regards,
KammaKhazi
